src/game/Snake.java

```diff
@@ -28,25 +26,13 @@ public void addHeadRectList() {
 
 	public void setStartPosition() {
 
-		int[] x = new int[16];
-		int[] y = new int[16];
-
-		int snakeStep = 25;
-
-		for (int i = 0; i < 16; i++) {
-
-			snakeStep += 25;
-
-			x[i] = snakeStep;
-			y[i] = snakeStep;
-
-		}
+		Random randomStartPosition = new Random();
 
 		int randomX = randomStartPosition.nextInt(16);
-		int randomy = randomStartPosition.nextInt(16);
+		int randomY = randomStartPosition.nextInt(16);
```

Comment:
I see that numbers for random generator here are a bit different than in FoodGenerator. Does it work properly ingame?

Reply:
I changed these code lines a bit.